server/engine/src/main/antlr4/org/eclipse/lsp/cobol/implicitDialects/cics/CICSParser.g4

```diff
@@ -268,9 +268,9 @@ cics_change_task: TASK (PRIORITY cics_data_value)? cics_handle_response?;
 
 /** CHECK ACQPROCESS / ACTIVITY / TIMER */
 cics_check: CHECK (cics_check_activity | cics_check_timer);
-cics_check_activity: (ACQPROCESS | ACTIVITY cics_data_value | ACQACTIVITY | COMPSTATUS cics_cvda | ABCODE cics_data_area |
+cics_check_activity: (ACQPROCESS | ACTIVITY cics_data_value)+ ( ACQACTIVITY | COMPSTATUS cics_cvda | ABCODE cics_data_area |
                      ABPROGRAM cics_data_area | MODE cics_cvda | SUSPSTATUS cics_cvda | cics_handle_response)+;
-cics_check_timer: TIMER cics_data_value cics_handle_response? STATUS cics_cvda cics_handle_response?;
+cics_check_timer: (TIMER cics_data_value cics_handle_response? STATUS cics_cvda cics_handle_response?)*;
```

Comment:
cics_handle_response 2 times??

I think this can be modified by ORing the commands and making TIMER mandatory in checkTimer(CICSParser.Cics_check_timerContext ctx).
This will give a better error message than "No viable alternative at input STATUS" when TIMER is missing.